logger.gd

```diff
@@ -53,8 +54,12 @@ class Logfile:
 			# TODO: Move directory creation to the function that will actually *write*
 			var err = dir.make_dir_recursive(base_dir)
 			if err:
-				print("[ERROR] [logger] Could not create the '%s' directory; exited with error %d." \
-						% [base_dir, err])
+				print(
+					(
+						"[ERROR] [logger] Could not create the '%s' directory; exited with error %d."
+						% [base_dir, err]
+					)
+				)
```

Comment:
That's not an improvement IMO.

Reply:
I agree. This happens because the line length is capped at 100. I could change the command to:
`gdformat --line-length=500 logger.gd`
This should solve this kind of formatting!